src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -200,7 +197,9 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdThe
       if (animateTo === animateFrom) {
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
-        lastDrawFrame = $$rAF(function animation(now) {
+        // Note that this shouldn't use use $$rAF, because it can cause an infinite loop
+        // in any tests that call $animate.flush.
+        $window.requestAnimationFrame(function animation(now) {
```

Comment:
We do lose all of that, but we could very easily replicated it if we needed.  The real problem is that we are trying to flush something that is by definition not meant to be flushed.  This is strictly animation code as far as I can tell.  It becomes (a) a lot more work and (b) a lot more fragile to try to keep it using $$rAF